features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/configuration/SlowFrameListenerConfiguration.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+package com.datadog.android.rum.configuration
+
+/**
+ * Configuration class for the slow frames listener, used to define thresholds for detecting
+ * slow frames, frozen frames, and ANRs (Application Not Responding) in the context of performance monitoring.
+ *
+ * @param maxSlowFramesAmount The maximum number of slow frame records to track. Default is 1000.
```

Comment:
Not quite clear from the customer point of view what does it mean: what is the record, how this amount is correlated with view event attributes, etc.

Reply:
updated doc 👍